nncf/common/pruning/utils.py

```diff
@@ -18,16 +18,20 @@
 
 import numpy as np
 
-from nncf.common.graph import NNCFGraph
-from nncf.common.graph import NNCFNode
-from nncf.common.graph.module_attributes import ConvolutionModuleAttributes
+from nncf.common.graph.graph import NNCFGraph
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.graph import NNCFNodeName
```

Comment:
The graph.py contents could be theoretically moved into __init__.py, but this would lead to the corresponding objects and related imports being imported each time any other sub-package of `nncf.common.graph` is imported, which IMO would lead to more risk of circular dependencies.

Reply:
Adjusted the `__init__.py` to import everything from `graph.py`, which removes the duplication of `.graph.graph` across the entire code.